### PR TITLE
PSMDB-777 Release notes PSMDB 3.6.21-10.0

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: source/release_notes/3.6.21-10.0.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+* :jirabug:`PSMDB-758`: Add mongobridge as part of percona-server-mongodb-server package
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PSMDB-766`: Redirect openldap debug messages to mongodb log
+* :jirabug:`PSMDB-755`: Add ldapDebug, ldapFollowReferrals and ldapConnectionPoolSizePerHost server parameters
```
Comment (akira-kurogane) [inline source/release_notes/3.6.21-10.0.rst]:
```suggestion
```

File: source/release_notes/3.6.21-10.0.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+Percona Server for MongoDB 3.6.21-10.0 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for `MongoDB 3.6.21 Community Edition <https://docs.mongodb.com/manual/release-notes/3.6/#nov-16-2020>`_.
+It supports MongoDB 3.6.21 protocols and drivers.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-758`: Add mongobridge as part of percona-server-mongodb-server package
+
```
Comment (akira-kurogane) [inline source/release_notes/3.6.21-10.0.rst:20]:
```suggestion
* :jirabug:`PSMDB-755`: Add ldapDebug, ldapFollowReferrals and ldapConnectionPoolSizePerHost server parameters

```

**Review by akira-kurogane — APPROVED** [review]
Hi Nastya.

Although PSMDB-755 was opened as a bug in the end it began the ticket where we decided the parameter names for the first time, so I think of it as the code change that added a user-visible feature. So I'd like to move that from the Bugs section to the Improvements section.

No other requests.

**Review by akira-kurogane — APPROVED** [review]
Thanks!

File: source/release_notes/3.6.21-10.0.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+:Date: November 26, 2020
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/3.6/install/index.html>`_
+
+Percona Server for MongoDB 3.6.21-10.0 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for `MongoDB 3.6.21 Community Edition <https://docs.mongodb.com/manual/release-notes/3.6/#nov-16-2020>`_.
+It supports MongoDB 3.6.21 protocols and drivers.
+
+Improvements
+================================================================================
+
```
Comment (igorsol) [inline source/release_notes/3.6.21-10.0.rst:16]: We also fixed PSMDB-376 in this release

**Review by igorsol — CHANGES_REQUESTED** [review]

**Review by igorsol — APPROVED** [review]
LGTM

**Pull request merged**